spec/src/modules/tracker.js

```diff
@@ -1957,10 +1957,10 @@ describe(`ConstructorIO - Tracker${bundledDescriptionSuffix}`, () => {
   describe('trackSearchResultsLoaded', () => {
     const term = 'Cat in the Hat';
     const requiredParameters = { numResults: 1337 };
-    const optionalParameters = { itemIds: [1, 2, 3] };
+    const optionalParameters = { item_ids: [1, 2, 3] };
     const legacyParameters = {
       ...requiredParameters,
-      customerIds: [1, 2, 3],
+      customer_ids: [1, 2, 3],
```

Comment:
Any reason why we changed these to be snake_cased? It should work for both camel + snake right? 🤔

Reply:
Not sure why I renamed them to be camel.